tdrs-frontend/.env.development

```diff
@@ -1,7 +1,7 @@
 REACT_APP_BACKEND_URL=http://localhost:8080/v1
-REACT_APP_TIMEOUT_TIME=6000
-REACT_APP_LOGOUT_TIME=10000
-REACT_APP_DEBOUNCE_TIME=1000
+REACT_APP_TIMEOUT_TIME=1000 * 60 * 20
+REACT_APP_LOGOUT_TIME=1000 * 60 * 3
+REACT_APP_DEBOUNCE_TIME=1000 * 30
```

Comment:
These times are in milliseconds so `1000 * 60 * 20` is 20 minutes, `1000 * 60 * 3` is 3 minutes and `1000 * 30` is 30 seconds. We annotated it this way as it's easier to read than a bunch of zeros.

Reply:
Great move!